examples/goext_example/example.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (C) 2017 NTT Innovation Institute, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"github.com/cloudwan/gohan/examples/goext_example/todo"
+	"github.com/cloudwan/gohan/extension/goext"
+)
+
+const (
+	entryCustomAction = "entry_cutom_action"
+)
+
+// Init is an entry point of this extension
+func Init(env goext.IEnvironment) error {
+	// Find the pre-loaded schema
+	schema := env.Schemas().Find("entry")
+
+	// Register runtime type for the schema (raw)
+	schema.RegisterRawType(todo.Entry{})
```

Comment:
Raw type is a type which can be serialized / deserialized from DB. It must contain proper db annotations. This line is necessary for gohan to allocate a proper runtime type (a struct) containing the data from DB and pass it to event handler so that in the event handler, user can read structure fields directly, not fields from some map.